### PR TITLE
[935][IMP] Add qualifications to employee

### PR DESCRIPTION
**Pull request by TimLai125**
Task #[935](https://www.quartile.co/web?debug=#id=935&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: l10n_jp_hr_employee/i18n/ja.po [hunk 1]
```diff
@@ -121,6 +121,11 @@ msgstr "アパート/ビル"
 msgid "Apartment/Building Furigana"
 msgstr "アパート/ビルフリガナ"
 
+#. module: l10n_jp_hr_employee
+#: model:ir.ui.view,arch_db:l10n_jp_hr_employee.view_employee_form
+msgid "Acquired Qualification"
+msgstr "持ち資格"
```
Comment (yostashiro) [inline l10n_jp_hr_employee/i18n/ja.po]:
```suggestion
msgid "Acquired Qualifications"
msgstr "取得済資格"
```

File: l10n_jp_hr_employee/models/hr_employee.py [hunk 2]
```diff
@@ -33,6 +33,12 @@ class HrEmployee(models.Model):
         compute='_compute_private_info_visible',
         string='Private Information Visibility',
     )
+    qualification_ids = fields.One2many(
+        'hr.qualification',
+        'employee_id',
+        string='Qualification',
```
Comment (yostashiro) [inline l10n_jp_hr_employee/models/hr_employee.py]:
```suggestion
        string='Qualifications',
```

File: l10n_jp_hr_employee/security/hr_security.xml [hunk 3]
```diff
@@ -15,21 +15,6 @@
...
-        </field>
-        <field name="groups" eval="[(4, ref('base.group_user'))]"/>
-    </record>
-    <record id="hr_qualification_manager_rule" model="ir.rule">
-        <field name="name">All Qualifications</field>
-        <field ref="model_hr_qualification" name="model_id"/>
-        <field name="domain_force">[(1,'=',1)]</field>
-        <field name="groups"
-               eval="[(4, ref('l10n_jp_hr_employee.group_employee_private_info_manage'))]"/>
-    </record>
```
Comment (yostashiro) [inline l10n_jp_hr_employee/security/hr_security.xml]: We can't do this given that the qualification model keeps attachments with sensitive info. What we may need to do is to create another model that casts over the qualification names and make a one2many relationship to that model.

File: l10n_jp_hr_employee/views/hr_employee_views.xml [hunk 4]
```diff
@@ -33,6 +33,11 @@
                     </div>
                 </div>
             </xpath>
+            <xpath expr="//page[@name='public']/group" position="after">
+                <group string="Qualification">
+                    <field name="qualification_ids" string="Acquired Qualification" widget="many2many_tags"/>
```
Comment (yostashiro) [inline l10n_jp_hr_employee/views/hr_employee_views.xml]:
```suggestion
                    <field name="qualification_ids" string="Acquired Qualifications" widget="many2many_tags"/>
```

**Review by yostashiro — CHANGES_REQUESTED** [review]

File: l10n_jp_hr_employee/security/hr_security.xml (hunk 3 above)
Comment (TimLai125) [reply l10n_jp_hr_employee/security/hr_security.xml]: Added a new compute field to `hr.employee` to retrieve the names of the qualifications.

**Review by yostashiro — APPROVED** [review]
LGTM!

**Pull request merged**